front/components/vaults/ConnectorDataUpdatedModal.tsx

```diff
@@ -43,13 +48,42 @@ export const ConnectorDataUpdatedModal = ({
                 Once synchronized, data will appear under{" "}
                 <em>"Connection Admin"</em>.
               </p>
-              <p>
-                Add data to <em>"Company Data"</em> for team-wide access or to a
-                specific vault for <em>restricted access</em> by some team
-                members.
+              <p className="font-bold">
+                Add data to{" "}
+                <Link
+                  className="cursor-pointer font-bold text-action-500"
+                  href={`/w/${owner.sId}/vaults`}
+                >
+                  <Hoverable onClick={() => {}}>Company Data</Hoverable>
+                </Link>{" "}
+                for team-wide access or to a specific vault for restricted
+                access by some team members.
```

Comment:
I'm also surprised by the some team members vs selected team members.

But I have no strong confidence here 👍 just in case